pom.xml

```diff
@@ -94,21 +94,22 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <module.name>${project.groupId}.${project.artifactId}</module.name>
     <sonatypeOssDistMgmtSnapshotsUrl>https://oss.sonatype.org/content/repositories/snapshots/</sonatypeOssDistMgmtSnapshotsUrl>
-    <maven.compiler.compilerVersion>1.7</maven.compiler.compilerVersion>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven.compiler.compilerVersion>1.8</maven.compiler.compilerVersion>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
What is the motivation here? 

If the issue is that Java 11 can't be used to target this version, the github action can be modified to use a lower java version.

Reply:
@elucash is happy to go with java 8+ support :) 

https://github.com/immutables/immutables/pull/1247#issuecomment-732422495